pycbc/inference/sampler_emcee.py

```diff
@@ -233,8 +247,12 @@ def write_results(self, fp, max_iterations=None):
             large enough to accomodate future data.
         """
         self.write_metadata(fp)
-        self.write_chain(fp, max_iterations=max_iterations)
-        self.write_likelihood_stats(fp, max_iterations=max_iterations)
+        self.write_chain(fp, start_iteration=start_iteration,
+                         end_iteration=end_iteration,
+                         max_iterations=max_iterations)
+        self.write_likelihood_stats(fp, start_iteration=start_iteration,
+                                    end_iteration=end_iteration,
+                                    max_iterations=max_iterations)
         self.write_acceptance_fraction(fp)
```

Comment:
Yeah, because emcee has a different definition of acceptance fraction. Rather than be the fraction of walkers who's jump proposal was accepted on each iteration (kombine), emcee returns the number steps accepted per walker (see [the emcee docs](http://dan.iel.fm/emcee/current/api/#emcee.EnsembleSampler) for details). The returned acceptance fraction therefore always has `len(number of walkers)`, regardless of the iteration.